bindata/network/multus-admission-controller/admission-controller.yaml

```diff
@@ -82,15 +82,24 @@ spec:
       - name: multus-admission-controller
         image: {{.MultusAdmissionControllerImage}}
         command:
-        - /usr/bin/webhook
-        args:
-        - -bind-address=0.0.0.0
-        - -port=6443
-        - -tls-private-key-file=/etc/webhook/tls.key
-        - -tls-cert-file=/etc/webhook/tls.crt
-        - -alsologtostderr=true
-        - -metrics-listen-address=127.0.0.1:9091
-        - -ignore-namespaces=openshift-etcd,openshift-console,openshift-ingress-canary,{{.IgnoredNamespace}}
+        - /bin/bash
+        - -c
+        - |-
+          set -euo pipefail
+{{- if .HyperShiftEnabled}}
+          while ! [[ -f /var/run/secrets/hosted_cluster/token ]]; do
+             echo "Waiting for client token"
+             sleep 2
+          done
+{{- end }}
```

Comment:
Should we eventually, after some time, exit with an error so that a possible problem is more evident to the outside world?
And this time probably needs to account for an offset delay from the time the file is created to when it appears in the mounted path.
Meaning I would not like this to always have restarts because the file is never going to be there at first but also not to be hung up forever hiding a possible problem.

Reply:
@jcaamano good point, updated it to eventually fail. This should at most take a couple of seconds